lib/linguist/languages.yml

```diff
@@ -3287,6 +3287,26 @@ Unified Parallel C:
   - .upc
   tm_scope: source.c
 
+Unity3D Asset:
+  type: data
+  ace_mode: yaml
+  color: "#ab69a1"
+  extensions:
+  - .anim
+  - .asset
+  - .mat
+  - .prefab
+  - .unity
+  tm_scope: source.yaml
+
+Unity3D Metadata:
```

Comment:
Why do you want to make a separate language for Unity3D Metadata? It looks like it has the same YAML syntax as the other files...